docs/src/contributing_websocket_http_api.md

```diff
@@ -2032,3 +2038,39 @@ Only the recipient can do this.
 
 `/feeds/u/user-name.xml?sort=Hot`
 
+### Images
+
+Lemmy forwards image requests to a locally running Pictrs.
+
+#### Get
+
+*Format and thumbnail are optional.*
+
+`GET /pictrs/image/{filename}?format={webp, jpg, ...}&thumbnail={96}`
```

Comment:
I just noticed now that you removed the check for allowed thumbnail sizes in our backend. Why? Someone could easily request thumbnails from 1..1000px for all images, and completely fill up the disk.

Reply:
We can add in a separate PR, cause we also have to choose which sizes other than original we want to support.